src/options/get.rs

```diff
@@ -275,4 +299,38 @@ pub mod tests {
 
         remove_file(git_config_path).unwrap();
     }
+
+    #[test]
+    fn test_get_themes_from_config() {
+        let git_config_contents = b"
+[delta \"dark-theme\"]
+    max-line-distance = 0.6
+    dark = true
+
+[delta \"light-theme\"]
+    max-line-distance = 0.6
+    light = true
+
+[delta \"light-and-dark-theme\"]
+    max-line-distance = 0.6
+    light = true
+    dark = true
+
+[delta \"not-a-theme\"]
+    max-line-distance = 0.6
```

Comment:
Tiny thing but can you use the `b#"..."#` string literal syntax that allows you not to escape the double quotes inside the string?

Reply:
👍 done.